src/python/WMCore/MicroService/Unified/MSOutputTemplate.py

```diff
@@ -0,0 +1,228 @@
+import time
+
+from copy import deepcopy
+from bson import Timestamp, ObjectId
+
+
+class MSOutputTemplate(dict):
+    """
+    A simple template to represent the objects stored in Mongodb for MSOutput
+    Its purpose should be to serve as a bidirectional representation of the
+    objects stored in Mongodb. The instances of this class should be used as a
+    buffer for both inbound (reading from the database) and outbound (writing to
+    database) documents. And it must have all the methods needed to set or update
+    various fields in the object. So that we assure the uniformity of all documents.
+
+    Template format:
```

Comment:
This template is not compliant with what has been discussed in the issue #9610, such as:
* "WorkflowName" reads "RequestName" over here. It's fine IMO, but this things need to be fed back to the issue where the requirements are laid out (update the issue if you keep it as is) 
* "RequestStatus" needs to go away
* "transferStatus" reads "isTaken" over here. Please update the GH issue with this parameter.
* "outputDatasets" reads "OutputDatasets" over here (capital). Please update the code here.

I see a clash between "isTaken" and "transferStatus", we don't need both arguments. So we need to pick one and use it for the consumer lookup. I don't have any preference, but the GH issue needs to reflect these changes.

Reply:
I will update the GH  issue. I basically changed the names so we can have as much overlap with Reqmgr as possible and also - I am actually automatically copying them from the initial document to the one which is about to be uploaded in MongoDB. look at this [1]
   * `RequestName` is an example of this exact thing
   * `RequestStatus` will go away maybe next month - this is put there temporarily wont be used for any login regarding the subscritopns
   * `OutputDatasets`   Same as `RequestName`
   * `transferStatus` does not overlap `isTaken` - one is the status of the Transfer at the Data Management system, the other is the flag if the document is taken from on of the threads of the MSOutput Module and is under evaluation - it is put there in order to avoid race conditions between the threads.
 
And overall about the initial capital letters - l would suggest that similar to Reqmgr we  capitalize them all here too.   

[1]
https://github.com/dmwm/WMCore/blob/eb2edafab1639a248d1771a1a344e0317710355e/src/python/WMCore/MicroService/Unified/MSOutputTemplate.py#L94-L101